src/google-apis/auth-client.ts

```diff
@@ -115,7 +116,7 @@ export class RequestHandler extends common.Service {
 
     var that = this;
     if (tryAuthenticate) {
-      this.authClient.getToken(function(err) {
+      this.authClient.getToken(function(err: {message: string}) {
```

Comment:
`function(err: Error)` would work.

Reply:
Would an arrow function work here?

Reply:
I will introduce arrow functions in an upcoming PR when I run `gts fix`.